packages/react-native/src/Authenticator/hooks/useFieldValues/utils.ts

```diff
@@ -39,6 +43,18 @@ export const getSanitizedRadioFields = (
       return false;
     }
 
+    if (
+      unverifiedContactMethods &&
+      Object.keys(unverifiedContactMethods).findIndex((key) => key === name) < 0
+    ) {
+      logger.warn(
+        `${name} is not supported. Available values are: ${Object.keys(
+          unverifiedContactMethods
+        )}.`
+      );
+      return false;
+    }
```

Comment:
We really just need to make sure that if a user passes a custom value to the `fields` prop of `VerifyUser` and it has a `name` that isn't one of the two keys that are allowed by `UnverifiedContactMethods` it's excluded. We don't need to add any additional props or types in this PR

Reply:
Wouldn't it be a better user experience to also print out what the accepted values are though? Just saying that filed x was excluded is not helpful enough imo.